src/Sylius/Component/Locale/Provider/LocaleCollectionProvider.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Locale\Provider;
+
+use Sylius\Component\Locale\Model\LocaleInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+
+final class LocaleCollectionProvider implements LocaleCollectionProviderInterface
```

Comment:
Just curious, why do you want to make distinction between `LocaleProvider` and `LocaleCollectionProvider`?

Reply:
BC policy forces me to do so. New method can be added to class/interface only in few scenarios (models, repositories).